system/jlib/jsecrets.cpp

```diff
@@ -306,6 +306,11 @@ class CVault
     time_t clientTokenExpiration = 0;
     bool clientTokenRenewable = false;
     bool verify_server = true;
+    unsigned retries = 3;
+    time_t retry_wait_seconds = 1;
```

Comment:
similar comment about variable names - normally we use camel case.

Reply:
It was broken, but I doubt this is currently used by anyone but me, although it could be useful at some point.  I'll create a separate JIRA/PR.